Maths/CoPrimeCheck.js

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
I believe algorithms files are supposed to be self contained.

Reply:
Sorry, I forgot it.